website/docs/docs/dbt-cloud-apis/sl-graphql.md

```diff
@@ -150,6 +150,60 @@ metricsForDimensions(
 ): [Metric!]!
 ```
 
+**Metric Types**
+
+```graphql
+Metric {
+  name: String!
+  description: String
+  type: MetricType!
+  typeParams: MetricTypeParams!
+  filter: WhereFilter
+  dimensions: [Dimension!]!
+  queryableGranularities: [TimeGranularity!]!
+}
+```
+
+```
+MetricType = [SIMPLE, RATIO, CUMULATIVE, DERIVED]
+```
+
+**Metric Type parameters**
+
+```graphql
+MetricTypeParams {
+  measure: MetricInputMeasure
+  inputMeasures: [MetricInputMeasure!]!
+  numerator: MetricInput
+  denominator: MetricInput
+  expr: String
+  window: MetricTimeWindow
+  grainToDate: TimeGranularity
+  metrics: [MetricInput!]
+}
+```
+
+
+**Dimension Types**
+
+```graphql
+Dimension {
+  name: String!
+  description: String
+  type: DimensionType!
```

Comment:
hey @WilliamDee, if a user uses these parameters, does this mean the `DimensionType!` field needs to be filled in by them using either `CATEGORICAL` or `TIME` as state below?

DimensionType = [CATEGORICAL, TIME]

Reply:
this is a return type so not an input type, so this is for them to retrieve certain values from the request